site-search/data.go

```diff
@@ -43,6 +43,17 @@ type Description struct {
 	Summary           string    `json:"summary"`
 	Title             string    `json:"title"`
 	Unit              string    `json:"unit,omitempty"`
+	Highlight         Highlight `json:"hightlight,omitempty"`
+}
+
+// Hightlight contains specfic metadata with search keyword(s) highlighted
+type Highlight struct {
+	Title           string    `json:"title"`
```

Comment:
All highlight fields are optional and hence json tag should contain omitempty (missing from both title and summary)